cromwell_setup/README.md

```diff
@@ -0,0 +1,13 @@
+# Cromwell Setup
+
+## Overview
+
+The notebooks in this directory are intended to provide examples setting up [Cromwell](https://cromwell.readthedocs.io/en/stable/) and running workflows with it. 
+
+## Getting started
+
+Start by running the [cromwell_examples.ipynb](https://github.com/DataBiosphere/terra-axon-examples/blob/main/cromwell_setup/cromwell_examples.ipynb). This notebook is intended to demonstrate how you can use the Cromwell engine on Terra to execute and manage workflows. Setup instructions are provided, along with examples of commands to submit workflows, check a job's status, list jobs, and examine output. This notebook also includes an example of submitting to a locally running Cromwell server if you have one running.
```

Comment:
s/Start by running/Run/
s/is intended to demonstrate/shows/
Maybe make the 'examples of commands to' list a bulleted list.
No need for 'if you have one running'